.circleci/config.yml

```
@@ -67,100 +58,81 @@ commands:
 
 jobs:
   build:
+    docker:
+      - image: planetariumhq/9c-launcher-circleci:git-6fa331fdef32d6f124d06b6c48d07e352cc11067
+        auth:
+          username: $DOCKER_USERNAME
+          password: $DOCKER_PASSWORD
+    resource_class: large
     parameters:
       os:
-        type: executor
-    executor: << parameters.os >>
+        type: string
     steps:
       - when:
           condition:
-            equal: [*windows, << parameters.os >>]
+            equal: [ windows, << parameters.os >> ]
```

```suggestion
            equal: [windows, << parameters.os >>]
```

Just bikeshedding formatting.

---

Probably you want to run the Prettier once on the file, like `yarn prettier --write .circleci/config.yml`